app/screens/sso/sso_with_redirect_url.tsx

```diff
@@ -62,23 +66,26 @@ const SSOWithRedirectURL = ({doSSOLogin, loginError, loginUrl, serverUrl, setLog
     const [error, setError] = useState<string>('');
     const style = getStyleSheet(theme);
     const intl = useIntl();
+    const codeVerifier = useMemo(() => generateRandomString(100), []);
```

Comment:
The memoization here means the verifier will be only generated once and could be reused in multiple SSO flows, right? Including across different servers? That's not good.

Reply:
No, is memoized only while the component is in memory, changing servers/ going back to the previous screen removes the verifier from memory